internal/pkg/service/buffer/cleanup/node_test.go

```diff
@@ -63,101 +65,185 @@ func TestCleanup(t *testing.T) {
 	assert.NoError(t, schema.Configs().Exports().ByKey(exportKey2).Put(export2).Do(ctx, client))
 	assert.NoError(t, schema.Configs().Exports().ByKey(exportKey3).Put(export3).Do(ctx, client))
 
-	createdAtRaw, _ := time.Parse(time.RFC3339, "2006-01-02T15:04:05+07:00")
-	createdAt := utctime.UTCTime(createdAtRaw)
-	timeNow := time.Now()
-
-	// Add file with a Closing state and created in the past - will be deleted
-	fileKey1 := key.FileKey{ExportKey: exportKey1, FileID: key.FileID(createdAt)}
-	file1 := model.File{
-		FileKey: fileKey1,
-		State:   filestate.Closing,
-		Mapping: model.Mapping{
-			MappingKey:  key.MappingKey{ExportKey: exportKey1, RevisionID: 1},
+	mapping := func(exportKey key.ExportKey) model.Mapping {
+		return model.Mapping{
+			MappingKey:  key.MappingKey{ExportKey: exportKey, RevisionID: 1},
 			TableID:     keboola.TableID{BucketID: keboola.BucketID{Stage: "in", BucketName: "test"}, TableName: "test"},
 			Incremental: false,
 			Columns:     []column.Column{column.ID{Name: "id", PrimaryKey: false}},
-		},
+		}
+	}
+
+	oldTimeRaw, _ := time.Parse(time.RFC3339, "2006-01-02T15:04:05+07:00")
+	oldTime := utctime.From(oldTimeRaw)
+	timeNow := time.Now()
+
+	// File 1 ----------------------------------------------------------------------------------------------------------
+	// State:
+	//     An old broken file AFTER EXPIRATION, in Closing state, the slice is Uploaded but not Imported.
+	// Expected result:
+	//     File, slice, records and statistics are deleted.
+
+	// Add file with a Closing state and created in the past
```

Comment:
V tomto commite je vylepseny detailny `cleanup` test.
Asi je to najlepsie pozriet cele.

Doplnil som tu file v stave `Imported`, na ktorom sa testuje rollup/ulozenie `sum` do export prefixu.